src/components/Footer.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PureComponent from 'react-pure-render/component';
+
+const styles = {
```

Comment:
I just tried it, and it throws an error when I put styles beforehand and won't display anything on localhost